docs/user/ppl/cmd/mvexpand.rst

```diff
@@ -0,0 +1,201 @@
+=============
+mvexpand
+=============
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Description
+============
+| The ``mvexpand`` command expands each value in a multivalue (array) field into a separate row, similar to Splunk's `mvexpand` command.
+| For each document, every value in the specified field is returned as a new row. This is especially useful for log analytics and data exploration involving array fields.
+
+| Key features of ``mvexpand``:
+- Expands array fields into multiple rows, one per value.
+- Supports an optional ``limit`` parameter to restrict the number of expanded values per document.
+- Handles empty, null, and non-array fields gracefully.
+- Works as a streaming/distributable command for performance and scalability.
+
+Version
+=======
+3.3.0
+
+Syntax
+======
+mvexpand <field> [limit=<int>]
+
+* **field**: The multivalue (array) field to expand. (Required)
+* **limit**: Maximum number of values per document to expand. (Optional)
```

Comment:
Clarify the behavior without limit specified? Either default limit or no limit. Is returned values in order or randomly?